src/store/getters/state-getters.ts

```diff
@@ -390,6 +390,8 @@ export const hasCorrectionDataChanged = (state: StateIF): boolean => {
     )
   }
   if (isFirmCorrectionFiling(state)) {
+    // *** FUTURE: should check the following:
+    // completing party (client error correction only)
```

Comment:
This is to along with the remaining changed in isCorrectionValid() below (lines 496+).